Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "fernet"
-version = "0.2.1"
-authors = ["Alex Gaynor <agaynor@mozilla.com>", "Ben Bangert <bbangert@mozilla.com>"]
+version = "0.2.2"
+authors = [
+    "Alex Gaynor <agaynor@mozilla.com>",
```

Comment:
This email address no longer works. Should either be my personal or removed.

Reply:
Let me go update that then. Is there a variation you'd prefer?